docs/synapse/devops.rst

```diff
@@ -6,7 +6,63 @@
 Synapse DevOps Guide
 ####################
 
-TBD - See :ref:`quickstart` for current docs.
+General
+=======
+
+TLS/SSL Deployments
+-------------------
+
+For production deployements, it is recommended that all services use the built in ``telepath`` SSL/TLS
+protocol.  You may deploy a service using TLS encryption by specifying a ``--telepath`` listen URL option, such
+as ``ssl://cortex.vertex.link/``.  If it is necessary to separate the socket bind address from the hostname,
+a hostname query parameter may be specified: ``ssl://0.0.0.0/?hostname=cortex.vertex.link``
+
+The client will also need to specify an SSL/TLS ``telepath`` URL such as: ``ssl://visi:passwd@cortex.vertex.link``.
+
+Once the ``ssl`` protocol is specified, the ``hostname``, either from a query parameter or from the URL's
```

Comment:
move 'the hostname' after 'network location,'

Reply:
wouldn't be grammatically correct.  lets discuss.